pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -1416,6 +1420,8 @@ public void doNamespaceBundleSplit() throws Exception {
 
     @Override
     public void stop() throws PulsarServerException {
-        // do nothing
+        loadReportCacheZk.shutdown();
+        availableActiveBrokers.shutdown();
+        scheduler.shutdown();
```

Comment:
I think we should handle [future scheduling task](https://github.com/yahoo/pulsar/blob/master/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java#L247) if it gets triggered by zk-watch?